token/transfer-hook/example/src/processor.rs

```diff
@@ -73,7 +73,9 @@ pub fn process_execute(
     Ok(())
 }
 
-/// Processes a [InitializeExtraAccountMetaList](enum.TransferHookInstruction.html) instruction.
+/// Processes a
+/// [InitializeExtraAccountMetaList](enum.TransferHookInstruction.html)
+/// instruction.
```

Comment:
I don't even remember if these links work, but we can leave as is

Reply:
Ugly, but ok!